guides/release/templates/binding-element-attributes.md

```diff
@@ -38,39 +38,41 @@ If `isAdministrator` is `false`, Handlebars will produce the following:
 <input type="checkbox">
 ```
 
-### Adding Other Attributes (Including Data Attributes)
+### Adding Attributes to Component Invocations
 
-By default, components only accept a limited number of HTML attributes.
-This means that some uncommon but perfectly valid attributes, such as `lang` or
-custom `data-*` attributes must be specifically enabled. For example, this template:
+Components in Ember can also accept attributes to be places on an element
+which they render. A common use for this might be `lang` or `data-*`
+attributes. For example, this template passes those two attributes to
+a link component:
 
 ```handlebars
-{{#link-to "photos" data-toggle="dropdown" lang="es"}}Fotos{{/link-to}}
+<LinkTo @route="photos" data-toggle="dropdown" lang="es">
+  Fotos
+</LinkTo>
 ```
 
-Will render the following HTML:
+Which will render render the following HTML:
 
 ```html
-<a id="ember239" class="ember-view" href="#/photos">Fotos</a>
+<a href="#/photos" data-toggle="dropdown" lang="es" id="ember239" class="ember-view">
+  Fotos
+</a>
 ```
 
-To enable support for these attributes, an attribute binding must be
-added for each specific attribute on the component.
-To do this, you can extend the appropriate components
-in your application. For example, for `link-to` you would create your own version
-of this component by extending
-[`Ember.LinkComponent`](https://www.emberjs.com/api/ember/release/classes/LinkComponent)
-
-```javascript {data-filename="app/components/link-to/component.js"}
-import LinkComponent from '@ember/routing/link-component';
-
-export default LinkComponent.extend({
-  attributeBindings: ['data-toggle', 'lang']
-});
-```
-
-Now the same template above renders the following HTML:
-
-```html
-<a id="ember239" class="ember-view" href="#/photos" data-toggle="dropdown" lang="es">Fotos</a>
-```
+Components can control which of their elements receive attributes.
+[Read more about how to customize a component's element](/components/customizing-a-components-element/).
```

Comment:
We'll want to use relative paths here so we don't hit snags when this file becomes part of `3.9`/`3.10` guides ...
```suggestion
[Read more about how to customize a component's element](../../components/customizing-a-components-element/).
```